Providers/Profile.swift

```diff
@@ -703,15 +703,22 @@ public class BrowserProfile: Profile {
 
         // Used as a task queue for syncing.
         private var syncReducer: AsyncReducer<EngineResults, EngineTasks>?
+        private var syncStatsReport: SyncStatsReport?
 
         private func beginSyncing() {
             notifySyncing(NotificationProfileDidStartSyncing)
+            if let account = profile.account {
+                syncStatsReport = SyncStatsReport(when: NSDate.now(), account: account, didLogin: false, why: "")
```

Comment:
The first thing that happens in a sync is we fetch `info/collections` to determine what's changed on the server. That fetch has the server timestamp in a header, and makes for an excellent symbolic beginning to a sync.